README.adoc

```diff
@@ -36,19 +36,6 @@ Existing tools such as `oc` are more operations-focused and require a deep-under
 |===
 |Language |Container Image |Supported Package Manager
 |*Node.js*
-|https://github.com/sclorg/s2i-nodejs-container[centos/nodejs-8-centos7]
-|NPM
-
-| |https://access.redhat.com/articles/3376841[rhoar-nodejs/nodejs-8]
-|NPM
-
-|
-|https://www.github.com/bucharest-gold/centos7-s2i-nodejs[bucharestgold/centos7-s2i-nodejs]
-|NPM
-
-|
-|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-8-rhel7[rhscl/nodejs-8-rhel7]
-|NPM
```

Comment:
Java seems to have moved from the language column to the container image column

Reply:
yeah taking a look

Reply:
done @mik-dass

Reply:
:+1: